plugins/column_types.cc

```diff
@@ -0,0 +1,121 @@
+/** column_types.cc
+    Jeremy Barnes, 27 March 2016
+    This file is part of MLDB. Copyright 2016 Datacratic. All rights reserved.
+
+    Code to determine how to store a column based upon its contents.
+*/
+
+#include "column_types.h"
+#include "mldb/sql/cell_value.h"
+#include "mldb/sql/expression_value.h"
+
+
+namespace Datacratic {
+namespace MLDB {
+
+
+/*****************************************************************************/
+/* COLUMN TYPES                                                              */
+/*****************************************************************************/
+
+/** This is an accumulator that keeps statistics on the types of values that
+    a column could have.  It's useful for knowing how to treat a column in
+    an algorithm.
+*/
+
+ColumnTypes::   
+ColumnTypes()
+    : numNulls(false), numIntegers(false),
```

Comment:
They are counters, and should be initialized with false.  Their values are important later on when we look for optimization opportunities in how we store the column's data.